Automattic/woocommerce-payments
# Remove unused import `NoticeOutlineIcon` to fix JS linter warning

#### Changes proposed in this Pull Request

This PR removes an unused import of `NoticeOutlineIcon` to fix the JS linter warning in the file `client/components/deposits-overview/next-deposit.tsx`. View a recent PR **Files changed** tab to view the warning (e.g. https://github.com/Automattic/woocommerce-payments/pull/7126/files).

This file does not need to be imported since the changes in #7078.

<img width="651" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/455adb9c-bafc-4273-8d19-15a3326544c0">


#### Testing instructions





* Navigate to this PR's **Files Changed** tab on the GH web interface.
* Ensure that the warning `'NoticeOutlineIcon' is defined but never used` does not appear.

-------------------

- [x] Covered with tests **N/A** this is a fix for a code linting issue
- [x] Tested on mobile (or does not apply) **N/A** this is a fix for a code linting issue

**Post merge**
